tests/conftest.py

```diff
@@ -43,3 +47,119 @@ def samplecat():
         xr_open_kwargs={"engine": "h5netcdf"},
     )
     return xs.DataCatalog({"esmcat": xs.catalog.esm_col_data, "df": df})
+
+
+@pytest.fixture
+def datablock_3d():
```

Comment:
@aulemahal I implemented this to support the need to test 3D data in `xscen`. Thoughts?